util/test/rdtest/testcase.py

```diff
@@ -333,6 +333,40 @@ def check_pixel_value(self, tex: rd.ResourceId, x, y, value, eps=util.FLT_EPSILO
 
         log.success("Picked value at {},{} in {} is as expected".format(x, y, res_details.name))
 
+    def check_pixel_sample_value(self, tex: rd.ResourceId, x, y, sample, value, eps=util.FLT_EPSILON):
```

Comment:
Could this be implemented as a `sample=0` default named parameter in `check_pixel_value`?

Reply:
Yeah, I'll add a new commit that does this.